tests/integration/targets/digital_ocean_cdn_endpoints/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
-endpoint: gh-ci-space.nyc3.cdn.digitaloceanspaces.com
-origin: gh-ci-space.nyc3.digitaloceanspaces.com
+# endpoint: gh-ci-space.nyc3.cdn.digitaloceanspaces.com
+# origin: gh-ci-space.nyc3.digitaloceanspaces.com
```

Comment:
Should these comments be removed?

Reply:
I should have probably put a note like "not sure why this Space started throwing 500x" (I need to get our folks to dig).